jenkins_jobs/modules/properties.py

```diff
@@ -234,19 +234,21 @@ def naginator_opt_out(registry, xml_parent, data):
     """yaml: naginator-opt-out
```

Comment:
Btw shouldn't this property be simply named "naginator"? I think that was the major point - we want to have some potential for future configuration of this plugin, right?

Reply:
OK my bad, the XML element is named NaginatorOptOutProperty so the name of this property is fine.